notebooks/ivf_flat_hpo_example.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        n_lists = trial.suggest_int("n_lists", math.sqrt(dataset_size)*0.5, math.sqrt(dataset_size)*1.5)
</pre><p>This looks great, but I think we should provide a wider range of values to try so that we can see the objective in action and hit our target recall/latency. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuvs/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>